devices/templatetags/devicetags.py

```diff
@@ -129,3 +130,10 @@ def as_nested_list(factvalue):
 @register.filter
 def splitstr(arg1, arg2):
     return arg1.split(arg2)
+
+
+@register.inclusion_tag('deletebutton.html')
+def deletebutton(viewname, *args):
```

Comment:
ah yes, I like that :)

But the template changes make the button look very differently.
![2018-09-07-113012_199x249_scrot](https://user-images.githubusercontent.com/5805185/45211081-de4dd180-b280-11e8-822d-92b9aabfce41.png)

Also, should that html go to `snippets/` ?

Reply:
done